spec/models/passenger_spec.rb

```diff
@@ -3,125 +3,161 @@
-
-        it 'sets the status to pending' do
-          @passenger.pending!
-          expect { @passenger.set_status('active') }
-            .to(change { @passenger.registration_status })
-        end
+    it 'does now sent the incorrect email' do
+      %i[notify_active notify_archived].each do |mail_method|
+        expect(PassengerMailer).not_to have_received(mail_method)
       end
     end
 
-    describe 'expiration_display' do
-      context 'permanent passenger' do
-        it 'returns nil' do
-          @passenger.update permanent: true
-          expect(@passenger.expiration_display).to be_nil
-        end
+    it 'delivers the mail' do
+      expect(mail).to have_received(:deliver_now)
+    end
+  end
+
+  describe '#set_status' do
+    subject(:call) { ->(status) { passenger.set_status(status) } }
+
+    context 'when setting the status to archived' do
+      let(:mail) { ActionMailer::MessageDelivery.new(PassengerMailer, :notify_archived) }
+
+      before do
+        allow(PassengerMailer).to receive(:notify_archived).and_return mail
+        allow(mail).to receive(:deliver_now).and_return true
       end
 
-      context 'temporary passenger' do
-        it 'returns the expiration date of the doctors note' do
-          date = @passenger.eligibility_verification.expiration_date
-          expect(@passenger.expiration_display).to eql date.strftime('%m/%d/%Y')
-        end
+      it 'sets the status to archived' do
+        expect { call['archived'] }.to change(passenger, :registration_status)
       end
-    end
 
-    describe 'temporary?' do
-      it 'returns true if the passenger is not permanent' do
-        expect(@passenger.temporary?).to be true
+      it 'sets the status and sends an email to the passenger' do
+        call['archived']
+        expect(mail).to have_received(:deliver_now)
       end
     end
 
-    describe 'not having a SPIRE' do
-      it 'is not valid' do
-        passenger = build(:passenger, spire: '')
-        expect(passenger).not_to be_valid
+    it 'sets the status to pending' do
+      passenger.active!
+      expect { call['pending'] }.to change(passenger, :registration_status)
+    end
+
+    it 'sets the status to active' do
+      passenger.pending!
+      expect { call['active'] }.to change(passenger, :registration_status)
+    end
+  end
+
+  describe '#expiration_display' do
+    subject(:call) { passenger.expiration_display }
+
+    it 'returns nil for a permanent' do
+      passenger.update permanent: true
+      expect(call).to be_nil
+    end
+
+    it 'returns the expiration date of the doctors note for temporary passengers' do
+      date = passenger.eligibility_verification.expiration_date
+      expect(call).to eq(date.strftime('%m/%d/%Y'))
+    end
+  end
+
+  describe '#temporary?' do
+    subject(:call) { passenger.temporary? }
+
+    it 'returns true if the passenger is not permanent' do
+      expect(call).to be(true)
+    end
+  end
+
+  describe 'not having a SPIRE' do
```

Comment:
I get you're just doing the minimum to appease rubocop but this also totally feels like an improper use of `describe`. I don't think I'll include this in requested changes, just wanted to note that in case it itches your brain too

Reply:
Yeah, I don't love this spec. But FWIW, I do have a branch that removes it... somewhere in my project to stop using Spire at all on this app.